manifests/params.pp

```diff
@@ -36,6 +36,11 @@
   $rdeck_home = '/var/lib/rundeck'
   $service_logs_dir = '/var/log/rundeck'
 
+  $rdeck_uuid = $::serialnumber ? {
```

Comment:
Yeah, it was a *real* pain before.  You could use `getvar` or `defined`, but `defined` was broken before puppet 3.7.5 IIRC (and I probably don't), and `getvar` broken before 4.13 of stdlib??  Something like that anyway.